[2025-01-26T17:08:28.960+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_d_pipeline_compass_reviews.group_ingestion.group_jobs_apple.APPLE_INGESTION_SANTANDER-WAY manual__2025-01-25T16:24:44+00:00 [queued]>
[2025-01-26T17:08:28.968+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_d_pipeline_compass_reviews.group_ingestion.group_jobs_apple.APPLE_INGESTION_SANTANDER-WAY manual__2025-01-25T16:24:44+00:00 [queued]>
[2025-01-26T17:08:28.968+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-01-26T17:08:28.989+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): group_ingestion.group_jobs_apple.APPLE_INGESTION_SANTANDER-WAY> on 2025-01-25 16:24:44+00:00
[2025-01-26T17:08:28.993+0000] {standard_task_runner.py:57} INFO - Started process 48 to run task
[2025-01-26T17:08:28.996+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_d_pipeline_compass_reviews', 'group_ingestion.group_jobs_apple.APPLE_INGESTION_SANTANDER-WAY', 'manual__2025-01-25T16:24:44+00:00', '--job-id', '327', '--raw', '--subdir', 'DAGS_FOLDER/dag_d_pipeline_compass_reviews.py', '--cfg-path', '/tmp/tmpiy9kwhxx']
[2025-01-26T17:08:28.997+0000] {standard_task_runner.py:85} INFO - Job 327: Subtask group_ingestion.group_jobs_apple.APPLE_INGESTION_SANTANDER-WAY
[2025-01-26T17:08:29.039+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_d_pipeline_compass_reviews.group_ingestion.group_jobs_apple.APPLE_INGESTION_SANTANDER-WAY manual__2025-01-25T16:24:44+00:00 [running]> on host f48abc7117a5
[2025-01-26T17:08:29.142+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='gacarvalho.contato@gmail.com' AIRFLOW_CTX_DAG_OWNER='gacarvalho' AIRFLOW_CTX_DAG_ID='dag_d_pipeline_compass_reviews' AIRFLOW_CTX_TASK_ID='group_ingestion.group_jobs_apple.APPLE_INGESTION_SANTANDER-WAY' AIRFLOW_CTX_EXECUTION_DATE='2025-01-25T16:24:44+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-25T16:24:44+00:00'
[2025-01-26T17:09:40.031+0000] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to success. Terminating instance.
[2025-01-26T17:09:40.032+0000] {process_utils.py:131} INFO - Sending 15 to group 48. PIDs of all processes in the group: [49, 48]
[2025-01-26T17:09:40.033+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 48
[2025-01-26T17:09:40.035+0000] {taskinstance.py:1632} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-01-26T17:09:40.037+0000] {logging_mixin.py:151} INFO - Erro inesperado: Task received SIGTERM signal
[2025-01-26T17:09:40.040+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=49, status='terminated', started='17:08:29') (49) terminated with exit code None
[2025-01-26T17:09:40.138+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=48, status='terminated', exitcode=0, started='17:08:28') (48) terminated with exit code 0
